acceptance/tests/api-gateway/api_gateway_test.go

```diff
@@ -126,7 +126,7 @@ func TestAPIGateway_Basic(t *testing.T) {
 
 			// On startup, the controller can take upwards of 1m to perform
 			// leader election so we may need to wait a long time for
-			// the reconcile loop to run (hence the 1m timeout here).
+			// the reconcile loop to run (hence the 2m timeout here).
```

Comment:
I'd be favor of dropping the implementation-specific bit and avoiding things falling out of sync
```suggestion
			// the reconcile loop to run (hence the timeout here).
```